addons/stock_landed_costs/wizard/purchase_config_settings.py

```diff
@@ -1,12 +1,10 @@
 # -*- coding: utf-8 -*-
 # Part of Odoo. See LICENSE file for full copyright and licensing details.
 
-from openerp import api, models
-from openerp.tools.translate import _
+from odoo import api, models, _
 
 
 class PurchaseConfigSettings(models.TransientModel):
-    _name = 'purchase.config.settings'
     _inherit = 'purchase.config.settings'
 
     @api.onchange('group_costing_method')
```

Comment:
As per your comment "its already migrated" so it means migrated code return single record and not migrated multiple :question: